### DISCUSSION TITLE
[Feature] Auto add to album session

### DISCUSSION THREAD

**Discussion by tmlmt** [immich-app/immich#11668] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The use case is a holiday trip, or any other social event: you’d want to keep adding newly taken pictures to the same album that you’ve started sharing with your family/friends/…

Current workflow on Mobile:
- Take picture
- Add to album set to auto backup
- Open Immich app
- Add photo to shared album

Proposed workflow:
- At the start of the holiday / event: 
    - Open Immich
    - Toggle « Auto add to album » and select the album in which to add newly uploaded pictures
- Then just:
    - Take picture 
    - Take picture 
    - …

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I think with a little bit of care, this will be possible with #1673 by creating a rule that just matches all newly uploaded assets.

**Reply by tmlmt** [reply]
Hmm I don't think so. What's proposed in that feature request is a rule-based _view_. And there's nothing _smart_ about an auto-add session, I would like to tell Immich "now, just everything new you detect should be added to this album".

- A range of dates is not enough. Say I go on a 2 week trip during which there a birthday. I would turn on the feature to auto-add first to the "Trip" album, and then temporarily switch to auto-add to the "Birthday" album, and then switch back to the previous one. 
- Tags are not yet in place as I understand (#1651). But then what this feature request would be is an auto-add tag to new uploads

**Reply by bo0tzz** [reply]
>  a rule-based view. 

Not quite. Rules will evaluate on events and take actions such as adding to an album.

**Reply by tmlmt** [reply]
Cool! Then looking forward to it🙂 when is it expected?

**Reply by bo0tzz** [reply]
We don't give ETAs, but we're working towards it.

**Reply by tmlmt** [reply]
Oh yeah, I had forgotten the golden rule. Looking forward to it then 😃
